src/docs-assembler/navigation.yml

```diff
@@ -58,76 +58,139 @@ toc:
   # In docs-content, I tried moving release-notes/index.md to release-notes/intro/index.md
   # and creating a release-notes/intro/toc.yml file that contains just that page,
   # but I still couldn't get it to work.
-  - toc: docs-content://release-notes/intro
+  - toc: release-notes
```

Comment:
I copied your changes from https://github.com/elastic/docs-content/pull/1030 over locally and built it with the config that exists in this PR. This is the result: 

<img width="1322" alt="Screenshot 2025-04-04 at 4 01 31 PM" src="https://github.com/user-attachments/assets/3ee1212a-fb22-49c1-b683-1469cafc720c" />

If you make the following change to this PR and do _not_ merge https://github.com/elastic/docs-content/pull/1030, the result will look like the image below the code suggestion:

```suggestion
  - toc: docs-content://release-notes/intro
```

<img width="1291" alt="Screenshot 2025-04-04 at 4 05 14 PM" src="https://github.com/user-attachments/assets/e0dd57f3-1333-4071-8543-5bd11ea494eb" />